### ISSUE TITLE
[telegram] Support ReplyKeyboardMarkup

### ISSUE THREAD

**Issue by misery** [openhab/openhab-addons#7279]
I like to send a ReplyKeyboardMarkup in Telegram. Please add support for that with a possible parameter.

Currently InlineKeyboardMarkup is supported.

**Comment by eric1905** [comment]
Do you think about something like that one: https://www.schimmer-media.de/forum/index.php?thread/1692-how-to-telegram-sub-men%C3%BCs-erstellen/

I just also came here to open an issue because I'd like to have this enhancement too. I came from iobroker and liked the possibility to show me a preconfigured keyboard and then send some commands to my telegram bot. So "thumbs up" for your issue.

**Comment by misery** [comment]
Yeah, that's what I mean. ReplyKeyboardMarkup is the underlying API for that.

I switch from ioBroker, too. So my bot misses this feature now. ;-)

**Comment by Necromancer1982** [comment]
Hi, 
I'm also looking for the custom Keyboard.
Used it in several µC-Projects and like it very much :-)

**Comment by eric1905** [comment]
@Skinah  I saw that you updated the library in https://github.com/openhab/openhab-addons/pull/11125 and mentioned this issue. But this does not help or is there a solution yet?

**Comment by Skinah** [comment]
It’s on a long list of things I would like to look at. But no code has been written. The library update was needed to get support for the feature so it now just needs to be implemented by someone.
